sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Internals
+{
+    /// <summary>
+    /// This partial class encapsulates the State.
+    /// This controls if we are in either the Ping or Post state.
+    /// </summary>
+    internal partial class Manager
+    {
+        private Action _callbackAction = () => { };
+
+        internal readonly State _state = new();
+
+        internal TimeSpan _period;
+        private readonly TimeSpan _pingPeriod = TimeSpan.FromSeconds(5);
+        private readonly TimeSpan _postPeriod = TimeSpan.FromSeconds(1);
+        // TODO: WILL NEED ADDITIONAL PERIODS DEFINED FOR BACKOFF.
+
+        private void InitializeState()
+        {
+            SetPingState();
+            Task.Run(() => Run(CancellationToken.None)); // TODO: USE AN ACTUAL CANCELLATION TOKEN
```

Comment:
Should we use a dedicated thread instead of Task. Not a blocker for this PR, you could leave a TODO.

Reply:
added acomment